app/src/main-process/menu/build-default-menu.ts

```diff
@@ -296,7 +297,13 @@ export function buildDefaultMenu(sharedProcess: SharedProcess): Electron.Menu {
   const showLogsItem: Electron.MenuItemOptions = {
     label: __DARWIN__ ? 'Show Logs in Finder' : 'S&how logs in Explorer',
     click() {
-      shell.showItemInFolder(getLogPath())
+      const logPath = getLogPath()
+      fileSystem.mkdir(logPath, function(err: NodeJS.ErrnoException): void {
```

Comment:
And while I think the next line is fine for when `err` is undefined, we should follow the signature for `fs.mkdir` and make it explicit that `err` may be undefined.